src/holochainClient.js

```diff
@@ -12,7 +12,7 @@ const CHAPERONE_SERVER = {
   test: 'http://198.199.73.20:8000/',
   production: 'https://chaperone.holo.host/'
 }
-const CHAPERONE_SERVER_URL = CHAPERONE_SERVER.production
+export const CHAPERONE_SERVER_URL = CHAPERONE_SERVER.test
```

Comment:
this should branch based on an environment variable right? We don't want `CHAPERONE_SERVER_URL = CHAPERONE_SERVER.test` in production.

Might be simplest to just have a `REACT_APP_CHAPERONE_SERVER_URL`  environment variable and not have these hard coded values at all.